packages/browser/src/tracing/request.ts

```diff
@@ -318,9 +322,10 @@ export function xhrCallback(
             type: 'xhr',
             'http.method': sentryXhrData.method,
             url: sentryXhrData.url,
+            'server.address': parsedUrl.host,
             [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.http.browser',
+            [SEMANTIC_ATTRIBUTE_SENTRY_OP]: 'http.client',
           },
-          op: 'http.client',
```

Comment:
Just out of curiosity, where is `span.op` set, if not here? I couldn't quite track it down myself.

Reply:
Under `SEMANTIC_ATTRIBUTE_SENTRY_OP`.

In OTEL spans there is no span op, so we set it as a semantic attribute `SEMANTIC_ATTRIBUTE_SENTRY_OP` (maps to `sentry.op` under the hood). When we translate from otel -> sentry when sending to sentry, we map `SEMANTIC_ATTRIBUTE_SENTRY_OP` to `op` on the sentry span interface.

Reply:
Roger, thanks!